R/model_income_tax.R

```diff
@@ -168,6 +172,12 @@ model_income_tax <- function(sample_file,
          paste0(absent_cols, collapse = "\n\t"), ".\n")
   }
   
+  #cg adjustment
+  if (!is.null(cg_discount_rate)) {
+    sample_file[, Taxable_Income := pmaxC(Taxable_Income - Net_CG_amt + Tot_CY_CG_amt * (1 - cg_discount_rate),
```

Comment:
You should model the change using only `Net_CG_amt` as the path from `Tot_CY_CG_amt` is

```
true capital gross -> Tot_CY_CG_amt -> losses -> discount -> Net_CG_amt
```

otherwise you're discounting the gross, not the net.